src/openforms/forms/models/form.py

```diff
@@ -603,6 +603,11 @@ def deactivate(self):
         logger.debug("Deactivating form %s", self.admin_name)
         self.save(update_fields=["active", "deactivate_on"])
 
+    def form_uses_component(self, component_type: str) -> bool:
```

Comment:
"form" in the name is a bit redundant, I will fix that in the feedback, if you agree, and this method will probably be updated since we may need to handle multiple types in other clients.

Reply:
I also think `uses_component` is better. Or maybe even `has_component`?